gpustack/utils/hub.py

```diff
@@ -81,6 +82,29 @@ def match_model_scope_file_paths(
     return matching_paths
 
 
+def get_model_file_size(model: Model, token: Optional[str] = None) -> int:
```

Comment:
There is already a function with this name, should we change the name of this function or merge them.

Reply:
The function in downloader counts all files to be downloaded. Here it counts all weight files. Update the name and comments to make it clearer.